client/indexer-service/README.md

```diff
@@ -0,0 +1,7 @@
+# Storage Hub Indexer Service
```

Comment:
As we add indexed fields/events/whatever, it will come very handy to have some docs here on what are the steps required to add "something else" to what is indexed.

Reply:
Also, it would be very helpful to have a "Run node as indexer" docs here.